ci/cron/src/Docs.hs

```diff
@@ -1,6 +1,27 @@
 -- Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 -- SPDX-License-Identifier: Apache-2.0
 
+
+-- | The docs cronjob runs through the following steps
+--
+-- 1. Download the list of available releases from GH releases.
+-- 2. Download the list of available releases from S3 based on the contents of
+--    `/versions.json` and `/snapshots.json`.
+-- 3. For each version that is in GH but not in S3 run through those steps:
+--    3.1. Check if the release has already been uploaded under `/$version/`. If so do nothing.
+--         We hit this for the split release process for 2.0 and newer.
```

Comment:
I may be missing something here - why do we hit it? As far as I can tell we currently don't put split releases on GitHub so they're not part of the list to start with.

We also don't upload their docs folder.

If this comment is aspirational, as in we expect this to be true once the assembly repo produces documentation, uploads it, and creates a release on GitHub, should we wait for all of that to be done before merging this PR?

Or maybe add a note to the assumption belox that they are not quite met yet?

Reply:
This is aspirational, I trust me more to implement this than I trust myself to remember to update this so I’d lean towards not marking this but happy to add a disclaimer if you prefer.

Once implemented, the assembly repo will push to both S3 and GH so we have a Github release, and it’s uploaded to S3 but we still need to update the top-level dir in this job once it’s marked stable.

Reply:
Ok, I think we're on the same page then. 👍